File: Assignment15/C/Program3.c
// PS - Accept number of rows and number of columns from user and display below pattern
// Row: 4
// Row: 3
// Output: 
/* 
4 	3 	2 	1 	
4 	3 	2 	1 	
4 	3 	2 	1 	
4 	3 	2 	1
*/

#include<stdio.h>

void Display(int Row, int Col) {
    int i = 0, j = 0;

    for(i = 1; i <= Row; i++){
        for(j = Col; j >= 1; j--){
            printf("%d \t",j);
        }
        printf("\n");
    }
}

int main() {
    int iValue1 = 0, iValue2 = 0;

    printf("Enter Rows \n");
    scanf("%d", &iValue1);

    printf("Enter Col \n");
    scanf("%d", &iValue2);

    Display(iValue1, iValue2);

    return 0;
}

File: Assignment38/Java/Program3.java
// PS - write java program which accepts number of rows and number of columns from user and display below pattern
// A	A	A	A	
// B	B	B	B	
// C	C	C	C	
// D	D	D	D

import java.util.*;

class Pattern
{
    public int Arr[][];

    public Pattern(int i, int j)
    {
        Arr = new int[i][j];
    }

    public void CharPattern()
    {
        char ch = 'A';
        for(int i = 0; i < Arr.length; i++)
        {
            for(int j = 0; j < Arr[i].length; j++)
            {
                System.out.print(ch+"\t");
            }
            ch++;
            System.out.println();
        }
    }
}

class Program3
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of rows");
        int iRows = sobj.nextInt();

        System.out.println("Enter number of columns");
        int iCols = sobj.nextInt();

        Pattern pobj = new Pattern(iRows, iCols);

        pobj.CharPattern();
    }
}